data/config/uniswap-wrapper.js

```diff
@@ -0,0 +1,11 @@
+const rpc = require('./uniswap-wrapper.rpc')
+const staging = require('./uniswap-wrapper.staging')
+const rinkeby = require('./uniswap-wrapper.rinkeby')
+const mainnet = require('./uniswap-wrapper.mainnet')
```

Comment:
shouldn't we add the usability config here as well?

Reply:
I didn't see it added to `court.js` and `presale.js`, that's why I didn't add it here either.
Also, I'm not sure using `usability` as `-n` (network) parameter would work out of the box, but it would definitely feel weird.
Finally, my idea was to use this config with `-c` param, as it's a one time use.